tests/Aspire.Hosting.Python.Tests/AddPythonAppTests.cs

```diff
@@ -1,6 +1,8 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+#pragma warning disable ASPIREHOSTINGPYTHON001 // Test for experimental feature
```

Comment:
(nit) can probably do this for the whole test assembly since it is testing the Python project.

Reply:
I applied your suggestion but this is failing locally. Can you notice a mistake?

Reply:
Don't use a suppressmessage attribute. But instead add it to the NoWarn property.

```xml
<PropertyGroup>
  <NoWarn>$(NoWarn);ASPIREHOSTINGPYTHON001</NoWarn>
```

Reply:
Interesting, that's what I was just working on. So the attribute actually doesn't work? We should update the docs then. Can do it as part of the related PR, not just for this diagnostics.